### PR TITLE
Ticket/aotech 6678 sync all fixes

### PR DESCRIPTION
**Pull request by phatsk**
This PR addresses syncing all objects from the UI. This changeset introduces a "queue" of objects to sync that will be shifted every time the current set of items is done syncing. This removes the need for blocking requests during an "All" sync.

### CODE REVIEW

File: assets/js/press-sync.js [hunk 1]
```diff
@@ -78,26 +67,40 @@ window.PressSync = ( function( window, document, $ ) {
...
+			}
 
-				// Round to two decimal places, mostly.
-				remaining_time = Math.round( remaining_time * 100 ) / 100;
+			// Round to two decimal places, mostly.
+			remaining_time = Math.round( remaining_time * 100 ) / 100;
 
-				progress_string += ' (' + [ 'Estimated time remaining:', remaining_time, time_left_suffix ].join(' ') + ')';
-			}
+			progress_string += ' (' + [ 'Estimated time remaining:', remaining_time, time_left_suffix ].join(' ') + ')';
```
Comment (jrfoell) [inline assets/js/press-sync.js:81]: We should switch these out of l10n strings from the plugin at some point (along with 'hours' and 'minutes'). The fact that this estimate exists is :100:

**Review by jrfoell — APPROVED** [review]
Thanks Zach, let me know if I should turn my comment into a feature request under "issues." Won't block this PR in the meanwhile.

**Review by prometheas — APPROVED** [review]
Looks good to me 💃

File: assets/js/press-sync.js (hunk 1 above)
Comment (phatsk) [reply assets/js/press-sync.js:81]: I definitely agree that we should move these to properly localized strings in the future.

**Pull request merged**